daily-challenges/2024-10/15/khanh5923_15.spec.js

```
@@ -0,0 +1,46 @@
+    .textContent();
+  const TGDD = convertNumber(priceTGDD);
+
+  // DDV
+  await page_DDV.goto(`https://didongviet.vn/`);
+  await page_DDV
+    .locator(`//div[contains(@class,'w-[28%]')]//input[@id='input-search']`)
+    .fill(`iPhone 16`);
+  await page_DDV
+    .locator(`//div[contains(@class,'w-[28%]')]//input[@id='input-search']`)
+    .press("Enter");
+  let priceDDV = await page_DDV
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'priceDDV' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)